looty/src/main/scala/looty/poeapi/PoeTypes.scala

```diff
@@ -161,13 +177,16 @@ object PoeTypes {
       def isCurrency = x.getFrameType == FrameTypes.currency && !isHideoutItem
       def isQuest = x.getFrameType == FrameTypes.quest
       def isMap = x.descrText.toOption.exists(_ contains "Travel to this Map")
-      def isFragment = vaalFragments(x.typeLine)
+      def isFragment = mapFragments(x.typeLine)
       def isFlask = x.descrText.toOption.exists(_ contains "Right click to drink.")
       def isJewel = x.descrText.toOption.exists(_ contains "Jewel Socket")
       def isDivinationCard = x.getFrameType == FrameTypes.divCard
 
       def isInSocket = x.socket.toOption.isDefined
 
+      def isProphecy = x.getFrameType == FrameTypes.prophecy
+      def isLeaguestone = x.descrText.toOption.exists(_ contains "place a Leaguestone")
```

Comment:
very clever, i wonder if there is something similar to this type of search that we could use for map fragments? then we can get rid of that list and won't hvae to update it everytime GGG adds new map fragments, i haven't played in so long that i don't have of the new stuff handy